test/n2z/N2Z_Regression.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ab819831",
+   "metadata": {},
+   "source": [
+    "# NetCDF to Zarr (N2Z) regression tests\n",
+    "\n",
+    "This Jupyter notebook runs a suite of regression tests against a sample collection for the NetCDF-to-Zarr service (N2Z). The *GPM IMERG Final Precipitation L3 Half Hourly 0.1 degree x 0.1 degree V06 ([GPM_3IMERGHH](https://search.uat.earthdata.nasa.gov/search?q=C1245618475-EEDTEST)) at GES DISC* is chosen for its size ~10Mb per granule. \n",
+    "\n",
+    "\n",
+    "## Set the Harmony environment:\n",
+    "\n",
+    "The cell below sets the `harmony_host_url` to one of the following valid values:\n",
+    "\n",
+    "* Production: <https://harmony.earthdata.nasa.gov>\n",
+    "* UAT: <https://harmony.uat.earthdata.nasa.gov>\n",
+    "* SIT: <https://harmony.sit.earthdata.nasa.gov>\n",
+    "* Local: <http://localhost:3000>\n",
+    "\n",
+    "The default value is for the UAT environment. When using this notebook there are two ways to use the non-default environment:\n",
+    "\n",
+    "* Run this notebook in a local Jupyter notebook server and change the value of `harmony_host_url` in the cell below to the value for the environment you require from the above list.\n",
+    "* Use the `run_notebooks.sh` script, which requires you to declare an environment variable `HARMONY_HOST_URL`. Set that environment variable to the value above that corresponds to the environment you want to test. That environment variable will take precedence over the default value in the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7b732c0",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "harmony_host_url = 'https://harmony.uat.earthdata.nasa.gov'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4979305a",
+   "metadata": {},
+   "source": [
+    "### Import required packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "93128635",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from harmony import Collection, Environment, Client, Request\n",
+    "from os import makedirs\n",
```

Comment:
Nit: I don't see `makedirs` used anywhere directly in the notebook. I'm guess, though, that was part of work to try and grab the output Zarr store as discussed in Slack.